src/nfv_test_api/v2/services/gnodeb.py

```diff
@@ -78,7 +78,7 @@ def kill(self, identifier: str) -> None:
             raise NotFound(f"No process running for gNodeB with nci {identifier}")
 
 
-class GNodeBService(BaseService[GNodeB, GNodeBCreate, GNodeBUpdate]):
+class GNodeBService(BaseService[GNodeB, GNodeBCreate, GNodeBCreate]):
```

Comment:
Why changing this? We might want to support update at some point, put is just not it

Reply:
Okay so the update is PATCH ?

Reply:
To keep things simple, yes